pkg/pool-stable/contracts/StablePoolAmplification.sol

```diff
@@ -193,6 +193,7 @@ abstract contract StablePoolAmplification is BasePoolAuthorization {
     function _isOwnerOnlyAction(bytes32 actionId) internal view virtual override returns (bool) {
         return
             (actionId == getActionId(this.startAmplificationParameterUpdate.selector)) ||
-            (actionId == getActionId(this.stopAmplificationParameterUpdate.selector));
+            (actionId == getActionId(this.stopAmplificationParameterUpdate.selector)) ||
+            super._isOwnerOnlyAction(actionId);
```

Comment:
Note: These are not in the diff in https://github.com/balancer-labs/balancer-v2-monorepo/pull/2059, but were present in the base already.